pkg/playground/react-demo-tooltip.jsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function() {
+    "use strict";
+
+    var React = require("react");
+
+    var Tooltip = require("cockpit-components-tooltip.jsx").Tooltip;
+
+    function showTooltipDemo (element, top_element) {
+        var markup = {
+            __html: "The user <b>junior</b> is not permitted to manage storage"
+        }
+        var tip = <span dangerouslySetInnerHTML={markup}></span>;
```

Comment:
why set the inner html instead of just writing the full markup in jsx?
```
<span>The user <b>junior</b> is not permitted to manage storage</span>
```